app/src/organisms/GripperWizardFlows/UnmountGripper.tsx

```diff
@@ -19,11 +26,32 @@ import unmountGripper from '../../assets/videos/gripper-wizards/UNMOUNT_GRIPPER.
 import type { GripperWizardStepProps } from './types'
 import type { GripperData } from '@opentrons/api-client'
 
+const GO_BACK_BUTTON_STYLE = css`
```

Comment:
Probably adding `TEXT` is better since this is passed to `StyledText` instead of `Btn`
```suggestion
const GO_BACK_BUTTON_TEXT_STYLE = css`
```